microcosm_pubsub/sentry.py

```diff
@@ -0,0 +1,60 @@
+from dataclasses import dataclass
+from os import environ
+from typing import Any, Optional
+from unittest.mock import MagicMock
+
+import sentry_sdk
+from microcosm.decorators import defaults
+from microcosm.object_graph import ObjectGraph
+
+
+@dataclass
+class SentryConfig:
+    dsn: Optional[str] = None
+    enabled: bool = False
+    client: Optional[Any] = None
+
+
+def before_send(event, hint):
+    """
+    Before sending a event to sentry scrub all values that are non-uuid data from the event
+
+    """
+    for exception in event["exception"]["values"]:
+        for frame in exception["stacktrace"]["frames"]:
+            for var_name, value in frame["vars"].items():
+                if var_name in ["self", "cls"]:
+                    continue
+                elif isinstance(value, str):
+                    if value.startswith("<function "):
+                        continue
+                    elif var_name.endswith("_id"):
+                        continue
```

Comment:
Maybe:
```
if all([
    value.startswith("<function "),
    var_name.endswith("_id"),
]):
    continue
```

Reply:
`any`, but also don't think this adds much to readability. Tend to avoid any/all as it evaluates all elements which in some cases is a waste losing the benefit of the early exit.